src/contact.ts

```diff
@@ -477,6 +477,37 @@ export class Contact implements Sayable {
    */
   public alias(empty: null): Promise<boolean>
 
+  /**
+   * GET / SET / DELETE the alias for a contact
+   *
+   * @param {(none | string | null)} newAlias , 
+   * @returns {(string | null | Promise<boolean>)}
+   *
+   * @example GET the alias for a contact
+   * ```ts
+   * const alias = contact.alias()  //@returns {(string | null)}
```

Comment:
Please use code to talk instead of comments, especially in examples section.

This will be better:

```ts
const alias = contact.alias()
if (alias === null) {
  console.log('You did not set any alias for contact ' + contact.name())
} else {
  console.log('You had set alias for contact ' + contact.name() + ':' + alias)
}
```